Makefile

```diff
@@ -178,7 +178,7 @@ lint:
 	SEARCH_PATH={$(ACTOR_PATH),$(LIBRARY_PATH)} && \
 	echo "Using search path '$${SEARCH_PATH}'" && \
 	echo "--- Running pylint ---" && \
-	bash -c "[[ ! -z $${SEARCH_PATH} ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint" && \
+	bash -c "[[ ! -z $${SEARCH_PATH} ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k --disable=no-absolute-import" && \
```

Comment:
Maybe add no-absolute-import to .pylintrc?

Reply:
Done, even py3k can be in pylintrc :-)